### PR TITLE
[sequence_task_3][MKryvonos] implementation

### PR DESCRIPTION
**Pull request by MKryvonos**

### CODE REVIEW

File: sequence_task_3/student/include/impl.h [hunk 1]
```diff
@@ -11,4 +12,19 @@
...
  * @warning YOU MUST USE STRING STREAMS AND STREAM ITERATORS!
  */
-std::string reverseWords(const std::string& input);
+std::string reverseWords(const std::string& input)
+{
+    std::istringstream iss(input);
+    std::vector<std::string> words(std::istream_iterator<std::string>{iss}, std::istream_iterator<std::string>());
+    std::string result = "";
+
+    std::reverse(words.begin(),words.end());
```
Comment (smay1613) [inline sequence_task_3/student/include/impl.h:21]: You can avoid std::reverse call here, if you'll use reverse iterators or front inserter.

File: sequence_task_3/student/include/impl.h [hunk 2]
```diff
@@ -11,4 +12,19 @@
...
+
+    std::ostringstream ss;
+    std::copy(words.begin(),words.end(), std::ostream_iterator<std::string>(ss, " "));
+    result = ss.str();
+
+    if (result.size() > 0)
+        result.erase(result.end() - 1);
```
Comment (smay1613) [inline sequence_task_3/student/include/impl.h:28]: Use appropriate pop_back() method.

**Review by smay1613 — CHANGES_REQUESTED** [review]

File: sequence_task_3/student/include/impl.h (hunk 2 above)
Comment (MKryvonos) [reply sequence_task_3/student/include/impl.h]: done

File: sequence_task_3/student/include/impl.h (hunk 1 above)
Comment (MKryvonos) [reply sequence_task_3/student/include/impl.h]: done

**Pull request closed without merging**